CORE/world_gadgets/router/router.rb

```diff
@@ -30,7 +30,7 @@ def find_current_page
   end
 
   def assert_application_is_on_page(page_class)
-    @world.logger.validation "Ensuring that application is on page [#{page_class}]"
+    @world.logger.debug "Ensuring that application is on page [#{page_class}]"
```

Comment:
So we are no longer spitting out the page validation in standard action modes? Going to pull this down to see myself, but curious how the logger then visually looks between pages - the page message was a nice way to visually split before.